taskcall/README.md

```diff
@@ -0,0 +1,60 @@
+# TaskCall
+
+## Overview
+
+TaskCall is a real-time incident response system that reduces system downtime by automating the response process. It continuously accepts feeds from monitoring tools to provide a comprehensive view of your system. It uses its on-call management and incident response mechanisms to assemble the right team and resolve incidents in the fastest possible time.
+
+TaskCall's integration allows Datadog users to bolster their operations by improving their incident awareness and simplifying the process in which they are handled. Incidents sync up bi-directionally between the two platforms. Once Datadog alerts are received in TaskCall, users will be able to systematically manage all incidents without being interrupted. Users will also benefit from improved impact visibility from dependency graphs and status dashboards. With better understanding of the state of your overall infrastructure efficient resolutions can be achieved.
+
+
+## Key Features
+
+- Correct on-call responders are notified as soon as an alert is received from Datadog.
+- Repeating alerts are automatically silenced to avoid interrupting the on-call responders when they are already looking at the incident.
+- The integration is bi-directional. Status and priority are synced up between Datadog and TaskCall.
+- Incidents are automatically resolved in TaskCall when alert conditions are no longer present.
+- The integration is available on all TaskCall subscription plans.
+
+
+## Setup
+
+The integration needs to be configured from both TaskCall and Datadog.
+
+### In TaskCall
+
```

Comment:
Please include a step for navigating to the Integration tile, going to the configure tab, and clicking Connect Accounts to kick off the OAuth flow.